OWNERS

```diff
@@ -1,28 +1,8 @@
 approvers:
   - abhi-g
-  - aronchick
-  - dansanche
-  - ewilderj
-  - jinchihe
   - jlewi
-  - kunmingg
-  - lluunn
-  - nickchase
-  - pdmack
   - richardsliu
   - sarahmaddox
-  - texasmichelle
-  - willingc
-  - dsdinter
-  - Ark-kun
-  - gaoning777
-  - hongye-sun
-  - IronPan
-  - neuromage 
-  - paveldournov
-  - qimingj
-  - rileyjbauer
-  - vicaire
   - joeliedtke
   - animeshsingh
   - rmgogogo
```

Comment:
Same above for rmgogogo

Reply:
Maybe part of Tech writers team?

Reply:
@animeshsingh This is a good question. `rmgogogo` is a lead engineer for Kubeflow Pipelines. I left them in as approver, because they're an active contributor and they're in a different timezone from most others, which is useful if a PR needs to be pushed through. (I'm the only other approver who's not on the US West Coast.)